mycroft/skills/media/__init__.py

```diff
@@ -0,0 +1,130 @@
+from os.path import join, dirname
+
+from adapt.intent import IntentBuilder
+from mycroft.skills import time_rules
+from mycroft.skills.core import MycroftSkill
+from mycroft.messagebus.message import Message
+from mycroft.configuration import ConfigurationManager
+
+from mycroft.util.log import getLogger
+import mycroft.skills.media.mopidy
+
+logger = getLogger(__name__)
+
+__author__ = 'forslund'
+
+
+class MediaSkill(MycroftSkill):
+    """
+        The MediaSkill class is a base class for media skills containing
+        vocabulary and intents for the common functions expected by a media
+        skill. In addition event handlers to lower volume when mycroft starts
+        to speak and raise it again when (s)he stops.
+
+        But wait there is one more thing! A small event emitter and matching
+        handler to stop media currently playing when new media is started.
+    """
+    def __init__(self, name):
+        super(MediaSkill, self).__init__(name)
+        self.isPlaying = False
+        config = ConfigurationManager.get()
+        self.base_conf = config.get('Media')
```

Comment:
If you look at the base `MycroftSkill`, you should be able to replace `self.base_conf` with just `self.config`, since that's defined in the base class as getting the section of the configuration file with the name of the current skill. Just replace the `Media` section of the config with `MediaSkill`.